docs/get_started.rst

```diff
@@ -1,17 +1,345 @@
+                    title: Deleted Metadata
+        repo_url: https://github.com/SFDO-Tooling/CumulusCI-Test
+    test:
+        name_match: %_TEST%
+
+
+
+Starting a New CumulusCI Project
+--------------------------------
+This section assumes that you have CumulusCI and ``git`` installed on your host.
+We first need to make a directory with our project's name, navigate into the directory, and initialize it as a git repository.
+
+.. code-block:: console
+
+    $ mkdir cci_project; cd cci_project
+
+    $ git init
+
+We now need to initialize our project as a CumulusCI project.
+
+
+
+Project Initialization
+^^^^^^^^^^^^^^^^^^^^^^
+Use the `cci project init` command from within a git repository to generate the initial version of a project's ``cumulusci.yml`` file.
+
+.. code-block:: console
+
+    $ cci project init
+    Name: MyRepoName
+    Package name: My Repo Name
+    Package namespace: mynamespace
+    Package api version [38.0]:
+    Git prefix feature [feature/]:
+    Git default branch [main]:
+    Git prefix beta [beta/]:
+    Git prefix release [release/]:
+    Test namematch [%_TEST%]:
+    Your project is now initialized for use with CumulusCI
+    You can use the project edit command to edit the project's config file
+
+.. code-block:: console
+
+    $ cat cumulusci.yml
+    project:
+        name: MyRepoName
+        package:
+            name: My Repo Name
+            namespace: mynamespace
+
+The newly created `cumulusci.yml` file is the configuration file for your project specific tasks, flows, and CumulusCI customizations.
+For more information regarding configuraiton, checkout our `project configuration <#TODO internal ref here>`_ section of the docs.
+You can add and commit it to your git repository:
+
+.. code-block:: console
+
+    $ git add cumulusci.yml
+    $ git commit -m "Initialized CumulusCI Configuration"
+
+
+
+Add Your Repo to GitHub
+^^^^^^^^^^^^^^^^^^^^^^^
+With your ``cumulusci.yml`` file committed, we now want to create a repository on GitHub for our new project and push our changes there.
+
+#. `Create a new repository <https://docs.github.com/en/free-pro-team@latest/articles/creating-a-new-repository>`_ on GitHub.
+#. At the top of your GitHub Repository's Quick Setup page, click the clipboard button to copy the remote repository URL.
+#. In your terminal, `add the URL for the remote repository <https://docs.github.com/en/free-pro-team@latest/articles/adding-a-remote>`_ to where your local repository will be pushed::
+
+    $ git remote add origin <remote_repository_url>
+
+#. Verify the remote was added successfullly with::
+
+    $ git remote -v
+
+#. `Push the changes <https://docs.github.com/en/free-pro-team@latest/github/using-git/pushing-commits-to-a-remote-repository>`_ in your local repository to GitHub::
+
+    $ git push -u origin master
+
+
+
+Convert an Existing Salesforce Project
+--------------------------------------
+
```

Comment:
What about an introductory paragraph here? Maybe something like

    Converting an existing Salesforce project to use CumulusCI may follow a number of different paths, depending on whether you're practicing the Org Development Model or the Package Development Model, whether or not you're already developing in scratch orgs, and the complexity of your project's dependencies on the org environment. We'll explore several examples of how an existing project can be built using CumulusCI. Your experience may vary. You're welcome to discuss project conversion in the CumulusCI Trailblazer group [link].